packages/admin/cms-admin/src/common/translation/AzureAITranslatorProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import { gql, useApolloClient } from "@apollo/client";
+import { ContentTranslationServiceProvider } from "@comet/admin";
+// import { GQLTranslateQuery, GQLTranslateQueryVariables } from "@src/common/translation/AzureAITranslatorProvider.generated";
+import React from "react";
+
+import { useContentScope } from "../../contentScope/Provider";
+import { GQLTranslateQuery, GQLTranslateQueryVariables } from "./AzureAITranslatorProvider.generated";
+
+export const AzureAITranslatorProvider: React.FunctionComponent = ({ children }) => {
```

Comment:
I believe we need to be able to conditionally enable/disable the provider via scope. So you need to pass through the enabled prop here.